client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.js

```diff
@@ -10,6 +10,7 @@ import { useSelector } from 'react-redux';
 import './DataFilesSearchbar.module.css';
 
 const DataFilesSearchbar = ({ api, scheme, system, className }) => {
+  const err = useSelector(state => state.files.error.FilesListing !== false);
```

Comment:
@rstijerina @jchuahtacc  Do we want to disable it when we are loading?  If so, do we want to do the same thing for the Add button.
```suggestion
  const err = useSelector(
    state =>
      state.files.loading.FilesListing === true ||
      state.files.error.FilesListing !== false
  );
```